src/maidsafe/launcher/ui/controllers/account_handler_controller.h

```diff
@@ -43,33 +49,38 @@ class AccountHandlerController : public QObject {
     CreateAccountView,
   };
 
-  AccountHandlerController(helpers::MainWindow& main_window, QObject* parent);
+  AccountHandlerController(MainWindow& main_window, QObject* parent);
+  ~AccountHandlerController() override;
+  AccountHandlerController(AccountHandlerController&&) = delete;
+  AccountHandlerController(const AccountHandlerController&) = delete;
+  AccountHandlerController& operator=(AccountHandlerController&&) = delete;
+  AccountHandlerController& operator=(const AccountHandlerController&) = delete;
 
   AccountHandlingViews currentView() const;
-  void setCurrentView(const AccountHandlingViews new_current_view);
-  Q_SIGNAL void currentViewChanged(AccountHandlingViews arg);
+  void SetCurrentView(const AccountHandlingViews new_current_view);
```

Comment:
No need to change if this is how you want the signature.  It's just the pass-by-const-value is unusual and disables the ability to move the arg in.  I'm more used to seeing pass-by-value or -by-const-ref.